packages/docusaurus-theme-classic/src/theme/Tabs/index.js

```diff
@@ -30,9 +34,7 @@ function Tabs(props) {
           </li>
         ))}
       </ul>
-      <div className="margin-vert--md">
-        {children.filter(child => child.props.value === selectedValue)[0]}
-      </div>
+      <div className="margin-vert--md">{selectedTabItem}</div>
```

Comment:
In order to achieve this goal, you can just use spread syntax:

```suggestion
      <div className="margin-vert--md">
        {[...children].filter(child => child.props.value === selectedValue)[0]}
      </div>
```

Reply:
Wow @lex111 would you elaborate on why this works? 

I tried 

```js
> [...{a: 1}]
VM4342:1 Uncaught TypeError: object is not iterable (cannot read property Symbol(Symbol.iterator))
    at <anonymous>:1:1
```

and it doesn't work for me in the console but it works in the component.

Reply:
Because there must be an array (`[...[{a: 1}]]`), i.e. if we have one child - `this.props.children` it will contain just single object, if several ones then an array of objects.

Reply:
Yeah I get that, my question is why in the case of a single object that syntax works? `[...{a: 1}]` is the equivalent of `children` being a single object but it's an error.

Reply:
Ah, I didn’t understand you for the first time,
It is because `children` is iterable object, so no errors.

Reply:
I see. Where in the docs does it say that `children` is an iterable object? I tried doing `console.log(children[Symbol.iterator])` but got `undefined`

Reply:
`children` is an opaque data structure, so it seems to me that you can rely on the current behavior, or use React.Children.toArray as originally proposed by the author.

Reply:
Nice catch, @lex111. Thank you.